web-common/src/features/sources/extract-table-name.spec.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  extractFileExtension,
+  extractTableName,
+  sanitizeEntityName,
+} from "@rilldata/web-common/features/sources/extract-table-name";
+import { describe, it, expect } from "vitest";
+
+function getVariations(
```

Comment:
This is still called "variations" which isn't very descriptive. Rename to "TestCases" and "generateTestCases"?